src/pages/ReimbursementAccount/ValidationStep.js

```diff
@@ -142,7 +151,7 @@ class ValidationStep extends React.Component {
                                 value={this.state.amount3}
                                 onChangeText={amount3 => this.setState({amount3})}
                             />
-                            {errorMessage && (
+                            {Boolean(errorMessage) && (
```

Comment:
Is there any reason for wrapping this in Boolean rather than `_.isEmpty`? That might be more consistent with other checks within the app

Reply:
No reason! It looks like it was already changed to `!_.isEmpty` somewhere else since I got a conflict there. Updated to use `!_.isEmpty` instead.